src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -297,9 +297,8 @@ func NonStopWriter(writers ...io.Writer) io.Writer {
 // It never fails and always return the length of bytes passed in
 func (t *nonStopWriter) Write(p []byte) (int, error) {
 	for _, w := range t.writers {
-		if n, err := w.Write(p); err != nil {
-			return n, err
-		}
+		// not checking for errors
+		w.Write(p)
```

Comment:
I bet this could have been from @uaalto's work on making sure to run errcheck. Maybe just do a ```_, _ := w.Write(p)``` or something to make errcheck happy @fffw?